test/unit/environment_test.rb

```diff
@@ -8,18 +8,9 @@ def setup
     end
   end
 
-  test "should have name" do
-    env = Environment.new
-    refute env.valid?
-  end
-
-  test "name should be unique" do
-    as_admin do
-      env = Environment.create :name => "foo"
-      env2 = Environment.new :name => env.name
-      refute env2.valid?
-    end
-  end
+  should validate_presence_of(:name)
+  should validate_uniqueness_of(:name)
+  should allow_value('new').for(:name)
```

Comment:
I don't know if this is equivalent.  Does it pick up changes to the friendly_ reserved words?